taichi/backends/cuda/codegen_cuda.cpp

```diff
@@ -80,7 +80,7 @@ class CodeGenLLVMCUDA : public CodeGenLLVM {
       }
 
       for (auto task : offloaded_local) {
-        TI_DEBUG("Launching kernel {}<<<{}, {}>>>", task.name, task.grid_dim,
+        TI_TRACE("Launching kernel {}<<<{}, {}>>>", task.name, task.grid_dim,
```

Comment:
IMO `TI_DEBUG` should not occur per-invocation, which make you can't see anything at all.
I use `TI_DEBUG` for printing shader source, and `TI_TRACE` for per-invocation ones, so that the shader source doesn't scroll away as long as `log_level=debug`.